docs/source/Plugins.rst

```diff
@@ -157,6 +157,20 @@ plugin (``json``) will be executed before the XUnit plugin (``xunit``).
 If the HTML result plugin is installed and enabled (``html``) it will
 be executed before both JSON and XUnit.
 
+Configuring the plugin execution order
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+On some circumstances it may be necessary to change the order in which plugins
+are executed.  To do so, add a ``order`` entry a configuration file section
+named after the plugin type.  For ``job.prepost`` plugin types, the section name
+has to be named ``plugins.job.prepost``, and it would look like this::
+
+  [plugins.job.prepost]
+  order = ['myplugin', 'jobscripts']
```

Comment:
(Not saying it should be included in this PR, not saying it should be implemented, I just think we should reconsider it with this in mind)